packages/database-mongo/__tests__/index.ts

```diff
@@ -725,6 +725,30 @@ describe('Mongo', () => {
     });
   });
 
+  describe('removeAllPasswordResetTokens', () => {
+    // eslint-disable-next-line jest/expect-expect
+    it('should not convert id', async () => {
+      const mongoOptions = new Mongo(databaseTests.db, {
+        convertUserIdToMongoObjectId: false,
+      });
+      await mongoOptions.setService('toto', 'twitter', { id: '1' });
```

Comment:
you need to call `removeAllPasswordResetTokens` here 😁

Reply:
🤦‍♂